frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/networks-tab-validation.ts

```diff
@@ -28,21 +28,13 @@ export const validateNetworks = (options: UpdateOptions) => {
   const validatedNetworks = networks.map(
     ({ networkInterfaceWrapper, networkWrapper, ...networkBundle }) => {
       const otherNetworkBundles = networks.filter((n) => n.id !== networkBundle.id); // to discard networks with a same name
-      const usedMultusNetworkNames = new Set(
```

Comment:
Glad to see this logic removed - it will simplify the code.

Reply:
Agreed =)